shared/team-building/index.tsx

```diff
@@ -132,15 +138,178 @@ const ContactsBanner = ({onRedoSearch, onRedoRecs}: {onRedoSearch: () => void; o
+        (this.props.recommendations &&
+          this.props.recommendations.findIndex(section => section.label === label)) ||
+        -1
+      if (sectionIndex >= 0 && Styles.isMobile) {
+        // @ts-ignore due to no RN types
+        ref.scrollToLocation({
+          itemIndex: 0,
+          sectionIndex,
+        })
+      }
+    }
+  }
+
+  _getRecLayout = (
+    sections: Array<SearchRecSection>,
+    indexInList: number
+  ): {index: number; length: number; offset: number} => {
+    let numSections = 0
+    let numData = 0
+    let length: 40 | 64 = 64
+    let currSectionHeaderIdx = 0
+    for (let i = 0; i < sections.length; i++) {
+      const s = sections[i]
+      if (indexInList === currSectionHeaderIdx) {
+        // we are the section header
+        length = 40
+        break
+      }
+      numSections++
+      const indexInSection = indexInList - currSectionHeaderIdx
+      if (indexInSection < s.data.length) {
+        // we are in this data
+        numData += indexInSection
+        break
+      }
+      // we're not in this section
+      numData += s.data.length
+      currSectionHeaderIdx += s.data.length + 1
+    }
+    const offset = numSections * 40 + numData * 64
+    return {index: indexInList, length, offset}
+  }
+
+  _listBody = () => {
+    const showRecPending = !this.props.searchString && !this.props.recommendations
+    const showLoading = !!this.props.searchString && !this.props.searchResults
+    if (showRecPending || showLoading) {
+      return (
+        <Kb.Box2 direction="vertical" fullWidth={true} gap="xtiny" style={styles.loadingContainer}>
+          <Kb.Icon style={Kb.iconCastPlatformStyles(styles.loadingIcon)} type="icon-progress-grey-animated" />
+          <Kb.Text type="BodySmallSemibold">Loading</Kb.Text>
+        </Kb.Box2>
+      )
+    }
+    if (!this.props.showRecs && !this.props.showServiceResultCount && !!this.props.selectedService) {
+      return (
+        <Kb.Box2
+          alignSelf="center"
+          centerChildren={true}
+          direction="vertical"
+          fullHeight={true}
+          fullWidth={true}
+          gap="tiny"
+          style={styles.emptyContainer}
+        >
+          <Kb.Icon
+            fontSize={Styles.isMobile ? 48 : 64}
+            type={serviceIdToIconFont(this.props.selectedService)}
+            style={Styles.collapseStyles([
+              !!this.props.selectedService && {color: serviceIdToAccentColor(this.props.selectedService)},
+            ])}
+          />
+          <Kb.Text center={true} type="BodyBig">
+            Enter a {serviceIdToLabel(this.props.selectedService)} username above.
+          </Kb.Text>
+          <Kb.Text center={true} type="BodySmall">
+            Start a Keybase chat with anyone on {serviceIdToLabel(this.props.selectedService)}, even if they
+            don’t have a Keybase account.
+          </Kb.Text>
+        </Kb.Box2>
+      )
+    }
+    if (this.props.showRecs && this.props.recommendations) {
```

Comment:
What about for search? No section list for search results?

Reply:
No, I didn't see anything relevant in designs for sections on search results.